ambari-server/src/main/java/org/apache/ambari/server/controller/AmbariManagementControllerImpl.java

```diff
@@ -2767,31 +2767,34 @@ private void createHostAction(Cluster cluster,
     List<OsSpecific.Package> packages =
```

Comment:
To be consistent with trunk add package_list only for INSTALL command.

    if (roleCommand.equals(RoleCommand.INSTALL)) {
      List<OsSpecific.Package> packages =
        getPackagesForStackServiceHost(ambariMetaInfo.getStack(stackId), serviceInfo, hostParams, osFamily);
      String packageList = gson.toJson(packages);
      commandParams.put(PACKAGE_LIST, packageList);
    }

Reply:
Fixed